docs/quickstart.rst

```diff
@@ -206,7 +206,7 @@ Create an application
     response = client.create_application(name='Example App', type='voice', answer_url=answer_url)
 
 Docs:
-`https://docs.nexmo.com/tools/application-api/api-reference#create <https://docs.nexmo.com/tools/application-api/api-reference#create?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library>`__
+`https://docs.vonage.com/tools/application-api/api-reference#create <https://docs.vonage.com/tools/application-api/api-reference#create?utm_source=DEV_REL&utm_medium=github&utm_campaign=python-client-library>`__
```

Comment:
This needs correct to point to the developer docs, this URL isn't valid. `docs.nexmo.com` is also just a redirect, so this should point to the current URL.